src/promptflow/tests/sdk_cli_test/e2etests/test_flow_local_operations.py

```diff
@@ -281,15 +282,21 @@ def test_flow_validation_failed(self, pf) -> None:
         flow_tools_path.unlink(missing_ok=True)
         validation_result = pf.flows.validate(flow=source)
 
-        assert "tool-meta" in validation_result
-        assert "Failed to load python module from file" in validation_result["tool-meta"].pop("prepare_examples.py", "")
-        assert "Meta file not found" in validation_result["tool-meta"].pop("summarize_text_content.jinja2", "")
-        assert validation_result == {
-            "inputs": {"url": {"value": {"type": ["Missing data for required " "field."]}}},
-            "tool-meta": {},
-            "outputs": {"category": {"value": {"type": ["Missing data for " "required field."]}}},
+        error_messages = copy.deepcopy(validation_result.error_messages)
+        assert "Failed to load python module from file" in error_messages.pop("nodes.2.source.path", "")
+        for yaml_path in [
+            "node_variants.summarize_text_content.variants.variant_0.node.source.path",
+            "nodes.1.source.path",
+        ]:
+            assert "Meta file not found" in error_messages.pop(yaml_path, "")
+
+        assert error_messages == {
+            "inputs.url.type": "Missing data for required field.",
+            "outputs.category.type": "Missing data for required field.",
         }
 
+        assert "line 22" in repr(validation_result)
```

Comment:
what's the str repr for a success validate result?

Reply:
```json
{
    "result": "Succeeded"
}
```